framework/include/variables/MooseVariableFV.h

```diff
@@ -457,12 +462,15 @@ class MooseVariableFV : public MooseVariableField<OutputType>
 
   void setActiveTags(const std::set<TagID> & vtags) override;
 
+  // Increment function for damping
+  const FieldVariableValue & increment() const { return _element_data->increment(); }
+
+protected:
```

Comment:
Does `isInternalFace` really need to be a protected method? This branch is causing a build failure on my local machine, as it is used in `FVAnisotropicDiffusion.C:61`.